### DISCUSSION TITLE
[Suggestion] Fake Window Decal

### DISCUSSION THREAD

**Discussion by Dwarf-LordPangolin** [tsunamayo/Starship-EVO#3944] [Suggestions]
For larger stations or ships, a lot of the interior space isn't going to have rooms that can be entered. This could potentially look a little bland from the outside, especially on the stations. Would it be possible to add a decal that looks like a window, sort of like the texture on this ship from X4: Foundations? I think it might help the game's world feel more alive.

![20210621012359_1](https://user-images.githubusercontent.com/33041786/122716089-70a3a200-d22f-11eb-85fb-220480ea4130.jpg)

(Apologies if this has already been suggested.)

**Comment by BigBadKangaroo** [comment]
I'd like to see a decal like that!

**Comment by Kaiser-Indrasil** [comment]
You can already make something exactly like this with the glow blocks, albeit on flat surfaces alone. That's why I think this is a good idea but instead of a prefab window-pattern glow decal, a glowing square and a glowing triangle decal would be very versatile. There's a lot of unnecessary glowing bits in sci fi designs which lots of us are very fond of (I know I am) and want to reproduce them.

**Comment by Dwarf-LordPangolin** [comment]
Yeah, I tried making some glow block windows, but it took a lot of time and didn't look right on angled surfaces. And putting it on a hinge would be way too much hassle.
![20210621011319_1](https://user-images.githubusercontent.com/33041786/122808799-f1e05080-d292-11eb-84f0-11d0ec0c4392.jpg)

That's a very valid point, Kaiser; a decal that can serve as any kind of glowy bit would be more versatile than a window-exclusive decal.

**Comment by Kaiser-Indrasil** [comment]
> That's a very valid point, Kaiser; a decal that can serve as any kind of glowy bit would be more versatile than a window-exclusive decal. 

That was in fact my original idea for the glowing bits.
Okay almost. I was thinking more of a glowing paint.
In theory, it would be way more versatile than introducing other block shapes, or even decal shapes. You could use existing decals and just paint them with the glow-emitting paint.

**Comment by Servomotor** [comment]
Ohh nooo...I love you Dwarf, but I CAN NOT STAND fake doors and windows. 

When I was a kid, we used to go to Disneyland in Southern California and Knotts Berry Farm. Both of them had false facades of doors and windows. It always bugged me that I could not open them or sometimes even look inside as the glass was painted black. 

I actually get mildly triggered when I use stickers to make a "hatch" or false door panel. I really hate doing it. I'd rather put a real door there and some pipes behind it. 

I can't stand that in no mans sky that in my freighter ship, I can't access the bottom sections of the ship. That kind of stuff drives me nuts. 

So, I am not really against your suggestion, but man I hate that kind of stuff. 
Greeble on the other hand does not bother me. In fact, I love it :D

**Comment by Servomotor** [comment]
In fact...now that we are on this topic. I'd like to see more varieties of hatch doors in game for cupboards, access panels etc. Anything that allows for me to open something without using a child entity. 

edit: And they should be clickable "F to interact" etc.

**Comment by ZachZent** [comment]
![image](https://user-images.githubusercontent.com/36980446/123088641-d84a2080-d3da-11eb-851e-95a2a9bfd7cf.png)
